extensions/arc/src/common/utils.ts

```diff
@@ -185,6 +185,22 @@ export async function promptAndConfirmPassword(validate: (input: string) => stri
 	return undefined;
 }
 
+/**
+ * Opens an input box
+ * @param validate A function that accepts value and returns an error message if it's invalid
+ * @returns Promise resolving to the value if it passed validation,
+ * or undefined if the input box was closed for any other reason
+ */
+export async function promptValue(header: string, prompt: string, validate: (input: string) => string): Promise<string | undefined> {
```

Comment:
This doesn't seem like something we really need a separate function for given that you can just call promptInputBox with the specified options directly (it's not doing any logic that would make this useful)